dtfs-central-api/api-tests/v1/tfm/facilities/tfm-amendments-get.api-test.js

```diff
@@ -0,0 +1,136 @@
+const wipeDB = require('../../../wipeDB');
+const app = require('../../../../src/createApp');
+const api = require('../../../api')(app);
+const CONSTANTS = require('../../../../src/constants');
+const { MOCK_DEAL } = require('../../mocks/mock-data');
+const aDeal = require('../../deal-builder');
+
+const mockUser = {
+  _id: '123456789',
+  username: 'temp',
+  roles: [],
+  bank: {
+    id: '956',
+    name: 'Barclays Bank',
+  },
+};
+
+const newFacility = {
+  type: 'Bond',
+  dealId: MOCK_DEAL.DEAL_ID,
+};
+
+const newDeal = aDeal({
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
+  additionalRefName: 'mock name',
+  bankInternalRefName: 'mock id',
+  editedBy: [],
+  eligibility: {
+    status: 'Not started',
+    criteria: [{}],
+  },
+});
+const createDeal = async () => {
```

Comment:
can we move these variables (except imports) inside the `describe` block? we should have done it for all other api tests.
but the idea is that these variables are block scoped

Reply:
done